packages/api/src/routes/helpers/request-logger.ts

```diff
@@ -4,6 +4,11 @@ import { getLocalStorage } from "@metriport/core/util/local-storage";
 import { analyzeRoute } from "./request-analytics";
 
 const asyncLocalStorage = getLocalStorage("reqId");
+const blackListedRoutes = [
+  "/internal/carequality/document-query/response",
+  "/internal/carequality/patient-discovery/response",
+  "/internal/carequality/patient-discovery/response",
```

Comment:
are we missing `document-retrieval`? if not - is this a duplicate?

Reply:
Good catch yea it was suppose to be document-retrieval